flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/PqStateChangeLogger.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.state.changelog;
+
+import java.io.IOException;
+
+interface PqStateChangeLogger<T> extends StateChangeLogger<T, Void> {
```

Comment:
If I guess correctly, Pq stands for priority queue, would it be fine to make it specific? It is difficult to guess what Pq stands for unless being familiar with the implmentaion.

Reply:
Yes, PQ=priority queue (and I wanted it to be symmetric to the "kv-" counterpart). 
I'll expand PQ as it creates confusion.